src/lib.rs

```diff
@@ -238,31 +256,50 @@ impl Builder {
     pub fn new() -> Builder {
         Builder {
             filter: filter::Builder::new(),
+            write_style: true,
             format: Box::new(|buf, record| {
                 let ts = buf.timestamp();
                 let level = record.level();
-                let level_color = match level {
-                    Level::Trace => Color::White,
-                    Level::Debug => Color::Blue,
-                    Level::Info => Color::Green,
-                    Level::Warn => Color::Yellow,
-                    Level::Error => Color::Red,
+                let mut level_style = buf.style();
+
+                match level {
+                    Level::Trace => level_style.set_color(Color::White),
+                    Level::Debug => level_style.set_color(Color::Blue),
+                    Level::Info => level_style.set_color(Color::Green),
+                    Level::Warn => level_style.set_color(Color::Yellow),
+                    Level::Error => level_style.set_color(Color::Red).set_bold(true),
                 };
 
-                let write_level = write!(buf.color(level_color), "{:>5}:", level);
-                let write_args = if let Some(module_path) = record.module_path() {
-                    writeln!(buf, " {}: {}: {}", ts, module_path, record.args())
+                if let Some(module_path) = record.module_path() {
+                    writeln!(buf, "{:>5} {}: {}: {}", level_style.value(level), ts, module_path, record.args())
                 }
                 else {
-                    writeln!(buf, " {}: {}", ts, record.args())
-                };
-
-                write_level.and(write_args)
+                    writeln!(buf, "{:>5} {}: {}", level_style.value(level), ts, record.args())
+                }
             }),
             target: Target::Stderr,
         }
     }
 
+    /// Initializes the log builder from the environment.
+    pub fn from_env<'a, E>(env: E) -> Self
+    where
+        E: Into<Env<'a>>
+    {
+        let mut builder = Builder::new();
+        let Env { filter, write_style } = env.into();
+
+        if let Ok(s) = env::var(&*filter) {
```

Comment:
Repeating this check is a bit nasty. I think we should add methods to `Env` to get an `Option` for each variable